systers_portal/meetup/views.py

```diff
@@ -7,3 +7,10 @@ class MeetupLocationAboutView(DetailView):
     """Meetup Location about view, show about description of Meetup Location"""
     model = MeetupLocation
     template_name = "meetup/about.html"
+
+
+class MeetupLocationMembersView(DetailView):
+    """Meetup Location members view, show members list of Meetup Location"""
+    model = MeetupLocation
+    template_name = "meetup/members.html"
+    paginate_by = 6
```

Comment:
6 is too small of a number. To find someone from a list of 500 members, that will require more than 80 pages. Change it to around 50 members per page.